templates/php.yml

```diff
@@ -1,26 +1,80 @@
 # PHP
-# Test and package your PHP project.
-# Add steps that run tests, save build artifacts, deploy, and more:
+# Build, package and deploy your PHP project.
```

Comment:
`Build, package and deploy your PHP project to Azure Linux Web App`
Can you please check this with PM?